framework/auth/campaigns.py

```diff
@@ -1,68 +1,98 @@
-import httplib as http
-
 import furl
+
 from werkzeug.datastructures import ImmutableDict
-from framework.exceptions import HTTPError
 
 from website import mails
 from website.settings import DOMAIN
 
-CAMPAIGNS = ImmutableDict({
+CAMPAIGNS = {
     'prereg': {
         'system_tag': 'prereg_challenge_campaign',
-        'redirect_url': lambda: furl.furl(DOMAIN).add(path='prereg/').url,
+        'redirect_url': furl.furl(DOMAIN).add(path='prereg/').url,
```

Comment:
`lambda` is no longer needed to solve circular import, see comments below for more information